src/fhir_mcp_server/server.py

```
@@ -289,8 +300,8 @@ async def search(
             async_resources: list[Any] = (
                 await client.resources(type).search(Raw(**searchParam)).fetch_raw()
             )
-            logger.debug("Async resources fetched:", async_resources) 
-            return async_resources
+            logger.debug("Async resources fetched:", async_resources)
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)